### PR TITLE
Expand node support

### PR DESCRIPTION
**Pull request by chawes13**
The purpose of this PR is to expand node support to allow for using the latest version of the [AWS Connect library](https://github.com/amazon-connect/amazon-connect-streams) (`>=12.0.0`).

I don't think this is a breaking change, since `@size-limit/file@3.0.0` is only compatible with node `>=10.0.0`. Let me know if I should take this as an opportunity to bump to 4 **or** if you want to keep support for 8

## Author Checklist

- [x] ~Add unit test(s)~
- [x] Update version in package.json
- [x] ~Update documentation (if necessary)~
- [x] ~Add story to storybook (if necessary)~
- [x] Assign dev reviewer

### CODE REVIEW

**Comment by dpikt** [comment]
Let's keep "support" for Node 8. Let's also add `^12` as an explicit version allowance to make this more future-proof.

**Comment by chawes13** [comment]
@dpikt Done!

File: package.json [hunk 1]
```diff
@@ -1,8 +1,8 @@
 {
   "name": "@launchpadlab/lp-components",
-  "version": "3.31.3",
+  "version": "3.31.4",
   "engines": {
-    "node": "^8.0.0 || ^10.13.0"
+    "node": "^8.0.0 || ^10.13.0 || ^11.0.0 || ^12.0.0"
```
Comment (dpikt) [inline package.json:5]: Do we want to support 11 since it's not LTS? We don't support 7 or 9 😏

**Comment by chawes13** [comment]
@dpikt I had to update tests to account for the sort algorithm changing in 11

File: package.json (hunk 1 above)
Comment (chawes13) [reply package.json]: Doneso

**Comment by chawes13** [comment]
Latest Travis report: https://travis-ci.org/github/LaunchPadLab/lp-components/builds/706629559

**Pull request merged**